### DISCUSSION TITLE
Welcome to grcengineering.github.io Discussions!

### DISCUSSION THREAD

**Discussion by p4gs** [grcengineering/grcengineering.github.io#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by griffeth** [comment]
:wave: Hello @p4gs! I've heard about GRC Engineering before (back in 2024) but I'm just now getting around to exploring it. I love it!!!

I work for GitHub on the Governance and Trust team and I would love to contribute to the effort here and learn from everyone who has already contributed.

One thought I have had for a long time is building a open-sourced guide for GRC professionals (on the "inside" implementing controls or on the "outside" as auditors/consultants) on how to think through securing/auditing various system components.

I don't know if something like that already exists or if there are resources like that spread throughout various system/application docs that we can consolidate into a single resource for people.

**UPDATE:** like I said, I just started exploring so I just ran across [this](https://github.com/grcengineering/community-security-controls?tab=readme-ov-file#solution-community-security-controls-framework-cscf) which sounds a little like what I'm describing! I'll keep exploring!!!
